### DISCUSSION TITLE
Deploy model on multiple GPU

### DISCUSSION THREAD

**Discussion by pl8787** [mosecorg/mosec#141] [Q&A] [answered]
How to deploy a copy of the model on different GPUs and tell the worker to schedule them?

**Comment by lkevinzc** [comment, marked as answer]
Hi @pl8787 ! You could pass `CUDA_VISIBLE_DEVICES` environment variable to the worker processes, so that each worker will occupy the specified GPU device and run in parallel to serve the requests.

See https://mosecorg.github.io/mosec/example/env/ for an example.
